packages/data-models/src/options/option-item.js

```diff
@@ -110,4 +110,13 @@ export default class OptionItem {
       }
     )
   }
+
+  /**
+   *
+   * @param {Array[Object]} valuesArr - Array[option's values]
+   */
+  uploadValuesFromArray (valuesArr) {
+    this.values = [...valuesArr]
+    this.defaultValue = this.values[0].value
```

Comment:
If the first value will always be the default one, maybe it's better to create a getter for it and avoid an assignment here?

Reply:
No, I don't agree. 
In an odinary case we upload defaultValue from the local json or remote API json.
And it could be defined not from value's list.
That's why I decided to save the current option definition workflow as it used now in both applications - `alpheios-core` and `alpheios-alignment-editor`

Reply:
@kirlat , I think , if you want to suggest some refactoring of Options definition I think it would be better to create a new issue  for that. And make such a refactoring in a separate task.